ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.ts

```diff
@@ -11,11 +13,15 @@ export class FaqSectionComponent {
     @Input() questions: string;
     @Input() questionsCount: number; /* don't pass the counter for displaying all questions */
     @Input() darkerBackground: boolean;
+    @Input() isColorectal: boolean;
     @ViewChild(MatAccordion) accordion: MatAccordion; /* Please, do not remove. It is used outside of the component */
 
-    constructor(private communicationService: CommunicationService) {}
+    constructor(private dialog: MatDialog) {}
 
     public openJoinMailingList(): void {
-        this.communicationService.openJoinDialog();
+        this.dialog.open(JoinMailingListComponent, {
```

Comment:
**_#optionally_**

We don't have `Join mailing list` link on Colorectal Page by design.
Perhaps, it's unnecessary stuff here.

Reply:
I think it's fine. We separate responcibilites. This component shouldn't be aware about if parent component's faq json text has or hasn't such item, just let's be here